src/Debug/Control.elm

```diff
@@ -35,26 +37,26 @@ import Time.Extra
 -}
 type Control a
     = Control
-        { currentValue : a
+        { currentValue : () -> a
         , allValues : () -> List a
-        , view : ControlView a
+        , view : () -> ControlView a
         }
 
 
 type ControlView a
     = NoView
-    | SingleView (() -> Html (Control a))
-    | FieldViews (List ( String, () -> Html (Control a) ))
+    | SingleView (Html (Control a))
+    | FieldViews (List ( String, Html (Control a) ))
```

Comment:
oh, hmm,  these were already lazy for computing the Html... I wonder if making the `view` field in the Control record is actually necessary?  I guess it probably is necessary?

Reply:
I think it's necessary unless you want to introduce a type describing how to render the controls:

```
type View =
    Checkbox | Dropdown
```

or something like that?